lib/ipa_test_kit/custom_groups/v1.0.0/ipa_smart_launch_group.rb

```diff
@@ -24,6 +27,14 @@ class IPASMARTLaunchGroup < Inferno::TestGroup
       group from: :smart_discovery_stu2 do
         run_as_group
 
+        config(
+          inputs: {
+            smart_auth_info: {
+              name: :standalone_smart_auth_info
```

Comment:
This also needs to be locked to public, and set to auth mode.

Reply:
This hasn't been done.

Reply:
this should now be fixed with the latest commit ([833cd7b](https://github.com/inferno-framework/ipa-test-kit/pull/40/commits/833cd7b5f72929e3fbc2a3e2e67875ffca32636c))

Reply:
I don't set auth mode at group level because this forces test 1.1 (SMART discovery) to get a client id and other auth mode inputs, which the current test kit does not require.

Reply:
It needs to be set to auth mode. They're going to need those inputs anyway to perform the launch, so I don't think it's a problem to have them in the discovery group too.

Reply:
fixed